manifests/pagerduty.pp

```diff
@@ -21,7 +21,8 @@
     type    => 'pipe',
     source  => 'puppet:///modules/sensu_handlers/pagerduty.rb',
     config  => {
-      teams => $teams,
+      teams       => $teams,
+      environment => $environment,
```

Comment:
So please let me know of the final agreement on this and I will make that change. @somic @solarkennedy @fessyfoo  even though it sounds not the best I am still ok with having environment in event (my first commit) and leaving handler confit the way it is.